src/frontend/apps/desk/src/pages/accessibility/index.tsx

```diff
@@ -0,0 +1,137 @@
+import { ReactElement } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import { Box, Text, TextStyled } from '@/components';
+import { PageLayout } from '@/core';
+import { useCunninghamTheme } from '@/cunningham';
+import { NextPageWithLayout } from '@/types/next';
+
+const Page: NextPageWithLayout = () => {
+  const { t } = useTranslation();
+  const { colorsTokens } = useCunninghamTheme();
+
+  return (
+    <Box>
+      <Box
+        as="h1"
+        $background={colorsTokens()['primary-100']}
+        $margin="none"
+        $padding="large"
+      >
+        {t('Accessibility')}
+      </Box>
+      <Box $padding={{ horizontal: 'large', vertical: 'big' }}>
+        <Text as="p" $display="inline">
+          <Text $weight="bold" $display="inline">
+            La Suite numérique
+          </Text>{' '}
+          s&apos;engage à rendre ses services numériques accessibles,
+          conformément à l&apos;article 47 de la loi n° 2005-102 du 11 février
+          2005.
+        </Text>
+        <Text as="h2" $margin={{ bottom: 'xtiny' }}>
+          Déclaration d&apos;accessibilité
+        </Text>
+        <Text as="p">Établie le 20 décembre 2023.</Text>
+        <Text as="p" $display="inline">
+          Cette déclaration d&apos;accessibilité s&apos;applique au site{' '}
+          <Text $weight="bold" $display="inline">
+            lasuite.numerique.gouv.fr
+          </Text>
+          .
+        </Text>
+        <Text as="h2" $margin={{ bottom: 'xtiny' }}>
+          État de conformité
+        </Text>
+        <Text as="p" $display="inline">
+          <Text $weight="bold" $display="inline">
+            lasuite.numerique.gouv.fr
+          </Text>{' '}
+          est non conforme avec le RGAA 4.1. Le site n&apos;a{' '}
+          <Text $weight="bold" $display="inline">
+            pas encore été audité.
+          </Text>
+        </Text>
+        <Text as="h2" $margin={{ bottom: 'xtiny' }}>
+          Amélioration et contact
+        </Text>
+        <Text as="p" $display="inline">
+          Si vous n&apos;arrivez pas à accéder à un contenu ou à un service,
+          vous pouvez contacter le responsable de lasuite.numerique.gouv.fr pour
+          être orienté vers une alternative accessible ou obtenir le contenu
+          sous une autre forme.
+        </Text>
+        <Text as="p" $display="inline">
+          <li>
+            E-mail :
+            <TextStyled
```

Comment:
one space is missing after ":"

Reply:
https://github.com/numerique-gouv/people/pull/207/commits/98545e5cf19fc253bad70fb61d63c539aafe263f